data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorConfig.java

```diff
@@ -32,6 +33,10 @@ public class KeyValueProcessorConfig {
     @NotEmpty
     private String fieldSplitCharacters = DEFAULT_FIELD_SPLIT_CHARACTERS;
 
+    @JsonProperty("include_keys")
+    @NotNull
+    private String[] includeKeys = DEFAULT_INCLUDE_KEYS;
```

Comment:
Please make this a `List<String>` to be easier to use. Also, update the `getIncludeKeys` method as well.

Reply:
Sure